src/bokeh/themes/__init__.py

```diff
@@ -92,8 +100,9 @@
 
 built_in_themes = {
     CALIBER       : Theme(json=_caliber.json),
+    CARBON        : Theme(json=_carbon.json),
     DARK_MINIMAL  : Theme(json=_dark_minimal.json),
-    LIGHT_MINIMAL : Theme(json=_caliber.json),
+    LIGHT_MINIMAL : Theme(json=_light_minimal.json),
```

Comment:
Please change the target branch to `branch-3.4`. The branch `bokeh:main` you selected was updated for the last time two years ago and this line was already corrected in this [commit](https://github.com/bokeh/bokeh/commit/c9c0adfae166d7efd1af3b08143af0e6fe449812#diff-c4ad4f86d32a852894df761b1ee4579d61d6b2e4ee43485a6e7729ad17c6a90e).

Please also add a screenshot of your example to show how it looks like.